website/docs/vortex-gui-docs.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Vortex GUI
+id: vortex-gui
+sidebar_position: 6
```

Comment:
We already have a page with `sidebar_position: 6`. I think this would fit well as `sidebar_position: 5` Can you change this and then find the ones with `sidebar_position: 5` and up and increase those by 1